src/main/java/org/spongepowered/api/entity/player/Player.java

```diff
@@ -144,4 +144,17 @@
      * @return The connection
      */
     PlayerConnection getConnection();
+
+    /**
+     * Kicks the player.
```

Comment:
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  adam@eq2.serveftp.net
    Domain eq2.co.uk has exceeded the max emails per hour (62/60 (103%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
Received: from github-smtp2-ext7.iad.github.net ([192.30.252.198]:51283 helo=github-smtp2a-ext-cp1-prd.iad.github.net)
	by web3.cov.yesit.co.uk with esmtps (TLSv1.2:DHE-RSA-AES256-GCM-SHA384:256)
	(Exim 4.84)
	(envelope-from <noreply@github.com>)
	id 1YJ5Au-00070m-3R
	for adam@eq2.co.uk; Wed, 04 Feb 2015 18:59:44 +0000
Date: Wed, 04 Feb 2015 10:59:44 -0800
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=github.com;
	s=pf2014; t=1423076384;
	bh=cMhV2joVoVSFVpYBUoB+OpU4g5QsM80TimBZf+0uTuo=;
	h=From:Reply-To:To:Cc:In-Reply-To:References:Subject:List-ID:
	 List-Archive:List-Post:List-Unsubscribe:From;
	b=c1QmJ6jqlDr1shmrcwZlAjitHJnr0AyqpAZPVGXvetODJKgd50UgscPD6tVlWGIpe
	 qXoNYup1HkRtOS/plVa63ep/+RzLkuPi1J+RxhUqSa/IC1erYlCacVT6EtwiRzr/t1
	 RKjR+ZRaF/MoLmStYoUx/ycNw9VMgUYJvs78BvBU=
From: Kenzie Togami <notifications@github.com>
Reply-To: SpongePowered/SpongeAPI <reply@reply.github.com>
To: SpongePowered/SpongeAPI <SpongeAPI@noreply.github.com>
Cc: Mumfrey <adam@eq2.co.uk>
Message-ID: <SpongePowered/SpongeAPI/pull/412/r24108875@github.com>
In-Reply-To: <SpongePowered/SpongeAPI/pull/412@github.com>
References: <SpongePowered/SpongeAPI/pull/412@github.com>
Subject: Re: [SpongeAPI] Bans API (#412)
Mime-Version: 1.0
Content-Type: multipart/alternative;
 boundary="--==_mimepart_54d26c2037528_dd83fdd081d72a0674649";
 charset=UTF-8
Content-Transfer-Encoding: 7bit
Precedence: list
X-GitHub-Sender: kenzierocks
X-GitHub-Recipient: Mumfrey
X-GitHub-Reason: comment
List-ID: SpongePowered/SpongeAPI <SpongeAPI.SpongePowered.github.com>
List-Archive: https://github.com/SpongePowered/SpongeAPI
List-Post: <mailto:reply@reply.github.com>
List-Unsubscribe: <mailto:unsub+001180bf5a9f8d7e382d74ecd0921f72b152642fbce5b0bb92cf0000000110ea2e2092a170ce016fdf4b@reply.github.com>,
 <https://github.com/notifications/unsubscribe/ABGAv3U9OiitCGPQB5tMcVWfD8wHCkpqks5nomOggaJpZM4DW2RI>
X-Auto-Response-Suppress: All
X-GitHub-Recipient-Address: adam@eq2.co.uk


----==_mimepart_54d26c2037528_dd83fdd081d72a0674649
Content-Type: text/plain;
 charset=UTF-8
Content-Transfer-Encoding: 7bit

> @@ -144,4 +144,17 @@
>       * @return The connection
>       */
>      PlayerConnection getConnection();
> +
> +    /**
> +     * Kicks the player.

Oh mail delivery.

---
Reply to this email directly or view it on GitHub:
https://github.com/SpongePowered/SpongeAPI/pull/412/files#r24108875
----==_mimepart_54d26c2037528_dd83fdd081d72a0674649
Content-Type: text/html;
 charset=UTF-8
Content-Transfer-Encoding: 7bit

<p>In <a href="https://github.com/SpongePowered/SpongeAPI/pull/412#discussion_r24108875">src/main/java/org/spongepowered/api/entity/player/Player.java</a>:</p>
<pre style='color:#555'>&gt; @@ -144,4 +144,17 @@
&gt;       * @return The connection
&gt;       */
&gt;      PlayerConnection getConnection();
&gt; +
&gt; +    /**
&gt; +     * Kicks the player.
</pre>
<p>Oh mail delivery.</p>

<p style="font-size:small;-webkit-text-size-adjust:none;color:#666;">&mdash;<br>Reply to this email directly or <a href="https://github.com/SpongePowered/SpongeAPI/pull/412/files#r24108875">view it on GitHub</a>.<img alt="" height="1" src="https://github.com/notifications/beacon/ABGAvzN1SJTnBjLSqARjo-4wtOsw2bpdks5nomOggaJpZM4DW2RI.gif" width="1" /></p>
<div itemscope itemtype="http://schema.org/EmailMessage">
  <div itemprop="action" itemscope itemtype="http://schema.org/ViewAction">
    <link itemprop="url" href="https://github.com/SpongePowered/SpongeAPI/pull/412/files#r24108875"></link>
    <meta itemprop="name" content="View Pull Request"></meta>
  </div>
  <meta itemprop="description" content="View this Pull Request on GitHub"></meta>
</div>

----==_mimepart_54d26c2037528_dd83fdd081d72a0674649--